### DISCUSSION TITLE
DLSS 3.0

### DISCUSSION THREAD

**Discussion by JxstNexo** [Blinue/Magpie#707] [Ideas]
Does it Have dlss 3?

**Comment by Blinue** [comment]
Supporting DLSS is impossible, because depth buffer and motion vector cannot be obtained.

**Reply by TenthSerpent** [reply]
what if it could read the outputs through something like Reshade which has access to both?

**Reply by Blinue** [reply]
This seems feasible, but it goes beyond the scope of Magpie.
